.github/scripts/end2end/install-kind-dependencies.sh

```diff
@@ -95,6 +95,23 @@ build_solution_base_manifests() {
     MANIFEST_ONLY=true $SOLUTION_BASE_DIR/build.sh
     sed -i 's/SOLUTION_ENV/default/g' $DIR/_build/root/deploy/*
     sed -i 's/MONGODB_STORAGE_CLASS/standard/g' $DIR/_build/root/deploy/*
+
+    # Limits and requests for MongoDB are computed based on the current system
+    # Detect total system RAM in GiB
+    TOTAL_RAM_GB=$(awk '/MemTotal/ {printf "%.0f", $2/1024/1024}' /proc/meminfo)
+    # Compute MongoDB settings based on the total RAM
+    MONGODB_WIREDTIGER_CACHE_SIZE_GB=$(awk "BEGIN {printf \"%.0f\", ${TOTAL_RAM_GB} * 0.335}")
+    MONGODB_MONGOS_RAM_LIMIT=$(awk "BEGIN {printf \"%.0fGi\", ${TOTAL_RAM_GB} * 0.165}")
+    MONGODB_SHARDSERVER_RAM_SAFE_LIMIT=$(awk "BEGIN {printf \"%.0fGi\", 2 * ${MONGODB_WIREDTIGER_CACHE_SIZE_GB}}")
+    MONGODB_SHARDSERVER_RAM_REQUEST=${MONGODB_WIREDTIGER_CACHE_SIZE_GB}Gi
+    MONGODB_MONGOS_RAM_REQUEST=$(awk "BEGIN {printf \"%.0fGi\", ${TOTAL_RAM_GB} * 0.033}")
+
+    # Replace values before deploying
+    sed -i "s/MONGODB_WIREDTIGER_CACHE_SIZE_GB/${MONGODB_WIREDTIGER_CACHE_SIZE_GB}/g" $DIR/_build/root/deploy/*
+    sed -i "s/MONGODB_MONGOS_RAM_LIMIT/${MONGODB_MONGOS_RAM_LIMIT}/g" $DIR/_build/root/deploy/*
+    sed -i "s/MONGODB_SHARDSERVER_RAM_SAFE_LIMIT/${MONGODB_SHARDSERVER_RAM_SAFE_LIMIT}/g" $DIR/_build/root/deploy/*
+    sed -i "s/MONGODB_SHARDSERVER_RAM_REQUEST/${MONGODB_SHARDSERVER_RAM_REQUEST}/g" $DIR/_build/root/deploy/*
+    sed -i "s/MONGODB_MONGOS_RAM_REQUEST/${MONGODB_MONGOS_RAM_REQUEST}/g" $DIR/_build/root/deploy/*
```

Comment:
you can actually use bash's math computation syntax : $(( 0.335 * ${TOTAL_RAM_GB} ))

we already rely on this, c.f. `count=$(($count + 1))` line 160, so we already must use bash (and not a generic /bin/sh as specified in the shebang)